eZ/Bundle/EzPublishCoreBundle/EventListener/CacheViewResponseListener.php

```diff
@@ -66,7 +66,14 @@ public function configureCache(FilterResponseEvent $event)
         }
 
         $response->setPublic();
-        if ($this->enableTtlCache) {
+
+        if (($maxAge = $view->getMaxAge()) !== null) {
+            $response->setMaxAge($maxAge);
+        }
+
+        if (($sharedMaxAge = $view->getSharedMaxAge()) !== null) {
+            $response->setSharedMaxAge($sharedMaxAge);
+        } elseif ($this->enableTtlCache) {
```

Comment:
Hm... Not really sure what to think about it. You can selectively disable it anyhow, but you wouldn't be able to selectively enable it.